wagtail/wagtailcore/models.py

```diff
@@ -1438,11 +1438,16 @@ class Meta:
         verbose_name_plural = _('page revisions')
 
 
+PAGE_PERMISSION_TYPES = [
+    ('add', _("Add"), _("Add/edit pages you own")),
```

Comment:
I use `"` for human-readable / translatable strings, and `'` for internal identifiers. (I *think* that's a standard convention, but it's possible that I just made it up :-) )